lib/validateSignature.ts

```diff
@@ -20,53 +18,45 @@ const hasValidSignature = (xml, cert, certThumbprint) => {
   const doc = parseFromString(xml);
   let signature =
     select(
-      doc,
-      "/*/*/*[local-name(.)='Signature' and namespace-uri(.)='http://www.w3.org/2000/09/xmldsig#']"
+      "/*/*/*[local-name(.)='Signature' and namespace-uri(.)='http://www.w3.org/2000/09/xmldsig#']",
+      doc
     )[0] ||
```

Comment:
Probably needs `?.[0]` here as expression can turn out to be `null`

Reply:
Can you make the changes please.